include/dem/pw_contact_force.h

```diff
@@ -83,6 +84,12 @@ class PWContactForce
     const ArrayView<const double> &particle_properties,
     const double &                 dt);
 
+  void
+  update_boundary_velocity(const DEMSolverParameters<dim> &dem_parameters);
+
+  Tensor<1, dim>
+  base_change(const Tensor<1, dim> tensor, const double theta);
+
```

Comment:
Remove from present PR, we can re-add them in the next one.

Reply:
I am afraid of removing them because i don't want to lose them. I will try